ddtrace/contrib/pytest/plugin.py

```diff
@@ -102,6 +104,11 @@ def pytest_runtest_protocol(item, nextitem):
         span.set_tag(test.SUITE, item.module.__name__)
         span.set_tag(test.TYPE, SpanTypes.TEST.value)
 
+        # Parameterized test cases will have a `callspec` attribute attached to the pytest Item object.
```

Comment:
Can we add a reference to the pytest codebase in case this ends up changing in the future?